course/patterns/footer.php

```diff
@@ -0,0 +1,173 @@
+          <p><a href="#"><?php echo esc_html__('Terms', 'course'); ?></a></p>
+          <!-- /wp:paragraph -->
+
+          <!-- wp:paragraph -->
+          <p><a href="#"><?php echo esc_html__('Privacy', 'course'); ?></a></p>
+          <!-- /wp:paragraph -->
+        </div>
+        <!-- /wp:group -->
+      </div>
+      <!-- /wp:column -->
+    </div>
+    <!-- /wp:columns -->
+  </div>
+  <!-- /wp:group -->
+
+  <!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"1.3rem"}},"layout":{"type":"constrained"}} -->
+  <div class="wp-block-group alignfull">
+    <!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} -->
+    <div class="wp-block-group alignfull">
+      <!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} -->
+      <div class="wp-block-group">
+        <!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained","contentSize":"1000px","justifyContent":"center"}} -->
+        <div class="wp-block-group">
+          <!-- wp:site-title {"textAlign":"left","isLink":false,"style":{"typography":{"letterSpacing":"0.02em","fontSize":"100px","lineHeight":"1"},"spacing":{"margin":{"top":"0px"},"padding":{"top":"0px"}}},"className":"footer-title","fontFamily":"sorts-mill-goudy"} /-->
+        </div>
+        <!-- /wp:group -->
+      </div>
+      <!-- /wp:group -->
+
+      <!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"0","padding":{"top":"0","right":"0","bottom":"0","left":"0"},"margin":{"top":"2.5rem"}}},"className":"separator","layout":{"type":"default"}} -->
+      <div class="wp-block-group alignfull separator" style="margin-top:2.5rem;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0">
+        <!-- wp:separator {"align":"full","style":{"spacing":{"margin":{"top":"0","bottom":"0"}}},"className":"is-style-default"} -->
+        <hr class="wp-block-separator alignfull has-alpha-channel-opacity is-style-default" style="margin-top:0;margin-bottom:0" />
+        <!-- /wp:separator -->
+      </div>
+      <!-- /wp:group -->
+    </div>
+    <!-- /wp:group -->
+    <!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"0"}}},"layout":{"type":"constrained","contentSize":""}} -->
+    <div class="wp-block-group alignfull" style="padding-top:0">
+      <!-- wp:paragraph {"align":"center","style":{"typography":{"letterSpacing":"0.02em"},"spacing":{"padding":{"bottom":"5rem","top":"var:preset|spacing|50"},"margin":{"top":"var:preset|spacing|70"}}},"fontSize":"x-small","fontFamily":"system"} -->
+      <p class="has-text-align-center has-system-font-family has-x-small-font-size" style="margin-top:var(--wp--preset--spacing--70);padding-top:var(--wp--preset--spacing--50);padding-bottom:5rem;letter-spacing:0.02em">
+        <?php
+        // Supports Footer Credit cusotmization on WPCOM and WoA sites.
+        $is_atomic = ( defined( 'ATOMIC_SITE_ID' ) && ATOMIC_SITE_ID );
```

Comment:
We try to avoid wpcom specific customization like this in themes, is there a way to remove this? I believe on simple sites we don't do anything like this and they are still able to show a custom footer.

Reply:
> We try to avoid wpcom specific customization like this in themes, is there a way to remove this?

@mikeyarce Would you mind chiming in on this as I believe you added the footer credit?

Also, while I have you here, any idea about the answers to these other questions? 🙂 

> Will the theme be available on the wpcom themes showcase as a free theme also?

> It looks like the theme is already in pub, to confirm there is no code there that will conflict with this version?

Reply:
The footer credit was customized because of how the WordPress.com Footer works - there is a specific regex in place that will not replace the custom footer we have on Sensei, which is why I opted to have a customization there.

If we want to remove our custom Sensei footer, then we don't need the specific wpcom customization.

> Will the theme be available on the wpcom themes showcase as a free theme also?

I'm not sure - I think it's up to the themes team what they want to do with it?  I would say it's probably best to make it a free theme.

>  It looks like the theme is already in pub, to confirm there is no code there that will conflict with this version?

The theme we commited to `pub` is a bit older, the only changes we would need to keep from the already committed one are the Headstart Annotations.

Reply:
> If we want to remove our custom Sensei footer, then we don't need the specific wpcom customization.

👍  That would be best to remove, then I think this is good to go.

Thanks!

Reply:
@mikeyarce Is this something you'd prefer to handle? Feel free to commit directly to this PR.

Reply:
Done in https://github.com/Automattic/themes/pull/6888/commits/e73455c1621677cb4b2a6e3fab3acde606c46984 😄